components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -47,100 +49,123 @@ object ErrorPages {
 enum class ErrorType(
     @StringRes val titleRes: Int,
     @StringRes val messageRes: Int,
-    @StringRes val refreshButtonRes: Int = R.string.mozac_browser_errorpages_page_refresh
+    @StringRes val refreshButtonRes: Int = R.string.mozac_browser_errorpages_page_refresh,
+    val imageId: Int? = null
 ) {
     UNKNOWN(
         R.string.mozac_browser_errorpages_generic_title,
         R.string.mozac_browser_errorpages_generic_message
     ),
     ERROR_SECURITY_SSL(
         R.string.mozac_browser_errorpages_security_ssl_title,
-        R.string.mozac_browser_errorpages_security_ssl_message
+        R.string.mozac_browser_errorpages_security_ssl_message,
+        imageId = 5
```

Comment:
Could the image IDs be strings or correspond to HTTP error codes? Right now you have to dig around to find what error an image corresponds to.

Reply:
The images correspond to more than one error type. I could name them as strings that roughly describe the image 🤔

Reply:
Works for me :)

Reply:
Done 😄